west.yml

```diff
@@ -128,7 +128,7 @@ manifest:
           compare-by-default: true
     - name: mcuboot
       repo-path: sdk-mcuboot
-      revision: c28768eb2aa7b68e1420b7e260f6139b5b019ebd
+      revision: v1.10.99-ncs1-rc1
```

Comment:
@nordicjm @de-nordic Is the mcuboot version correct? The very first sentence in the mcuboot readme file says:
> This is MCUboot version 1.**11**.0-dev

Reply:
I too the version from the Release planning page. If it is comfirmed there I will update here

Reply:
This is problematic for me, because while there is version 1.10.x, there is no version 1.11.x so this references series of tags that do not even exist yet. There is though tag 2.0.0-rc1.

Reply:
Question is whether we follow something from documentation or configuration management artifact that allows to find given version series?

Reply:
> configuration management artifact

What would that be?

I mean, It's up to you eventually, as project maintainers, to tell which version shall be used for the release. For me it's just a bit confusing, that there seems to be no single source-of-truth for the mcuboot version. Here's for example another reference, to 1.11.99 this time:
https://github.com/nrfconnect/sdk-mcuboot/blob/main/boot/zephyr/VERSION

Reply:
i will go with v1.10.99-ncs1-rc1

Reply:
I support the decision as user may be able to access commits with the v1.10.x tags but will not be able to query any commits given v.1.11.x tag. Basically you can not diff against v1.11.
I think there is a problem with the version in the documentation that it has not been given a real tag referencing any code.

Reply:
> > configuration management artifact
> 
> What would that be?
> 
> I mean, It's up to you eventually, as project maintainers, to tell which version shall be used for the release. For me it's just a bit confusing, that there seems to be no single source-of-truth for the mcuboot version. Here's for example another reference, to 1.11.99 this time: https://github.com/nrfconnect/sdk-mcuboot/blob/main/boot/zephyr/VERSION

Tag would be one. Pinned version.